_posts/2019-07-1-foreman-community-newsletter---june-2019.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title: Foreman Community Newsletter (June 2019)
+date: 2019-07-1 13:10:18
+author:
+ name: Rahul Bajaj
+ uri: rabajaj_
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+ We're back! 3 months of goodies, including the 1.22 release, my new beginning as
+ community lead, conference roundup, and community survey results!
+---
+
+Welcome back to the Foreman Newsletter! We kinda slipped up in May, sorry
+about that, but we're back with 3 months of news, gossip, and releases for you!
+
+### Joining the community team
+
+At the start of March, I made a decision to join th Foreman community team.
+I have been running the community demos on youtube as you might have already
+heard and recently I have been working with the community to get the foreman
+documentation more structured and readable. I would speak more about it in the
```

Comment:
I'd phrase this as; "I will speak more about about the documentation efforts in the upcoming demos and blog to keep you updated on the progress" because there where too many "it" in the sentence.